feature-libs/order/root/model/replenishment-order.model.ts

```diff
@@ -1,63 +1,4 @@
-import {
```

Comment:
We should remove the whole file. The goal was to keep things clear and simple like we stated in the ticket description.
Lets discuss.